src/mlpack/tests/math_test.cpp

```diff
@@ -631,6 +631,8 @@ BOOST_AUTO_TEST_CASE(SparseShuffleTest)
     data(0, i) = i;
     labels[i] = i;
   }
+  // This appears to be a necessary workaround for an Armadillo 8 bug.
+  data *= 1.0;
```

Comment:
Interesting, perhaps we should update the matrix build, maybe we already test against armadillo 8?

Reply:
I'm in the process of updating the matrix build, but it's not ready yet.  I haven't isolated the Armadillo bug yet, but I'll either open an issue or send a fix to Conrad.